inbm-lib/inbm_lib/trtl.py

```diff
@@ -193,7 +193,7 @@ def image_pull_public(self, image: str, reference: Optional[str], file_name: Opt
     @staticmethod
     def _send_password(cmd: str, password: str) -> Tuple[str, Optional[str], int]:
 
-        p = Popen(shlex.split(str(cmd)), stdout=PIPE, stdin=PIPE, stderr=PIPE)
+        p = Popen(shlex.split(str(cmd)), shell=False, stdout=PIPE, stdin=PIPE, stderr=PIPE)
```

Comment:
**<h2>$\color{#15F4EE}Miscellaneous$</h2>**
**Suggestion:** Ensure that the 'shell' parameter is set to False when using Popen to prevent shell injection vulnerabilities.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        p = Popen(shlex.split(str(cmd)), shell=False, stdout=PIPE, stdin=PIPE, stderr=PIPE)
```